env-prod-edgeware.json

```diff
@@ -0,0 +1,15 @@
+{
+  "USERLIB_ADDRESS": "0x84Ab6aa9d22E4C83456bb8FA57FcdA042e3efE1c",
```

Comment:
This values must be in `test` config. We are using their mainnet but this is contracts for our test environment.